### PR TITLE
Fix #6911

### LINKED ISSUE #6911: Diona nymph inhand inconsistency
The shoulder the nymph is sitting on changes depending on whether the holder is facing north/south or east/west.

image
image

Both were taken with the nymph in the left hand.

### PR DESCRIPTION
**Pull request by TheEliminator16**
- Fixes a sprite issue with diona nymphs riding on shoulders

fix #6911 just in case the title wasn't enough

### CODE REVIEW

**Comment by TheEliminator16** [comment]
Jesus christ, merge conflicts again. I think it's the changelog's fault

**Comment by TheEliminator16** [comment]
Will re-add changelog as soon as travis finishes

**Comment by TheEliminator16** [comment]
are you kidding me

**Comment by TheEliminator16** [comment]
god dammit

**Comment by TheEliminator16** [comment]
YES

File: html/changelogs/TheEliminator16.yml [hunk 1]
```diff
@@ -1,7 +0,0 @@
-author: TheEliminator16
-delete-after: True
-changes:
-  - bugfix: Made brown gloves properly use the on mob sprite.
```
Comment (ghost) [inline html/changelogs/TheEliminator16.yml:4]: HOLD ON

File: html/changelogs/TheEliminator16.yml [hunk 2]
```diff
@@ -1,7 +0,0 @@
...
-  - bugfix: Gave the detached limb sprite all four directions, just as god intended.
```
Comment (ghost) [inline html/changelogs/TheEliminator16.yml:5]: THIS SHOULD HAVE BEEN AUTOMATICALLY MERGED

File: html/changelogs/TheEliminator16.yml [hunk 3]
```diff
@@ -1,7 +0,0 @@
...
-  - bugfix: Fixed the bedhead hairstyle having certain pixels in one direction but not the opposite direction.
```
Comment (ghost) [inline html/changelogs/TheEliminator16.yml:6]: HENCE THERE'D BE NO REASON FOR CONFLICTS

File: html/changelogs/TheEliminator16.yml [hunk 4]
```diff
@@ -1,7 +0,0 @@
...
-  - bugfix: Fixed an issue with a voice changer set to look like a facehugger becoming invisible when pushed off the face.
```
Comment (ghost) [inline html/changelogs/TheEliminator16.yml:7]: PANIC

**Comment by gbasood** [comment]
okay here's what you want to do

``` bash
git checkout Bleeding-Edge html/changelogs/TheEliminator16.yml
```

should revert any changes you made to the file since you opened the branch

**Comment by 9600bauds** [comment]
Give your changelogs different filenames to prevent this
stuff like 9600bauds_dependingonthetimeofdaythefrenchgoeitherway.yml and 9600bauds_allowmeanaccuratedistantexaminationofmylittleautismalgroveasopposedtothecurrentdistantexaminationwhichisplaguedbyconcerningexistentialquestions.yml

**Comment by ghost** [comment]
The changelogs DO have different filenames. I think the changelog compiler is broken again.

**Comment by clusterfack** [comment]
Nothing is wrong with the changelog compiler he's using an old branch instead of a new bleeding edge branch

**Comment by TheEliminator16** [comment]
So is the number of commits too high to merge this?

**Comment by clusterfack** [comment]
I would prefer if you rebased this to bleeding edge and squashed it desu

**Pull request closed without merging**